package.json

```diff
@@ -146,6 +146,7 @@
     "url": "https://github.com/stevesims/govuk-frederic.git"
   },
   "author": "Steve Sims",
+  "contributors": ["Tarandeep Singh Chauhan", "Gavin Orland", "Toby Brancher"],
```

Comment:
I'd be inclined to have the `author` as `The govuk-frederic authors` and then list us all as contributors

We should also add Alasdair Mcleay, Mark Chambers, Jonathan Sheldrake, and Simon Moore to the contributors list as well

Reply:
👍  Have added these in alphabetical order.